src/main/java/io/lighty/yang/validator/formats/ConsoleLine.java

```diff
@@ -56,13 +56,19 @@ protected void resolveFlag(SchemaNode node, SchemaContext context) {
             }
             if (node instanceof ChoiceSchemaNode) {
                 qNames.remove(qNames.size() - 1);
-                if (context.findDataTreeChild(qNames).get().isConfiguration()
+                //TODO Rework this orElseThrow to schemaInterferenceStack when upstream will be current ODL master
+                if (context.findDataTreeChild(qNames)
+                        .orElseThrow(() -> new NullPointerException("Data tree child " + qNames.toString()
+                                + " not found.")).isConfiguration()
```

Comment:
```
context.findDataTreeChild(qNames)
                        .orElseThrow(() -> new NullPointerException("Data tree child " + qNames.toString()
                                + " not found.")).isConfiguration()
```
these constructs are too complicated to be in `if` clause - it is hard to read
simplify, for example, by extracting  to variable or reformat